app/controllers/mixins/vm_show_mixin.rb

```diff
@@ -167,6 +167,6 @@ def set_session_data
   end
 
   def breadcrumb_name(model)
-    ui_lookup_for_model(model || self.class.model.name).pluralize
+    ui_lookup(:models => model || self.class.model.name)
```

Comment:
@martinpovolny the original code `ui_lookup_for_model(model || self.class.model.name).pluralize` had `pluralize` at the end.  As a reviewer, I am expecting the refactored code `ui_lookup(:models => model || self.class.model.name)` to have the equivalent via `:plural => true`.  Why do you say that it is not introduced via this PR?